doc/chart.md

```diff
@@ -0,0 +1,12 @@
+# SCF
```

Comment:
I don't think this file belongs here. Especially because it contains instructions specific to the directory it was before. Also, there are still references to SCF here.

Reply:
Fixed with commit 50709e88bd8c8904bda167df86e25e06e621d083